agent/consul/leader_test.go

```diff
@@ -2126,6 +2126,8 @@ func TestLeader_EnableVirtualIPs(t *testing.T) {
 		t.Skip("too slow for testing.Short")
 	}
 
+	virtualIPVersionCheckInterval = 50 * time.Millisecond
```

Comment:
Normally I guess we'd want to reset this back to the original value after the test ends, but I'm guessing in this case it won't matter too much. Might be good to do just in case the pattern is copied in the future.

```suggestion
        orig :=  virtualIPVersionCheckInterval
	virtualIPVersionCheckInterval = 50 * time.Millisecond
	t.Cleanup(func() { virtualIPVersionCheckInterval = orig })
```